### PR TITLE
1) Rename and reorder Building an API - Part 1

### PR DESCRIPTION
**Pull request by beccaelenzil**
Rename and reorder Building an API Part 1 to correspond to branches being built in Hello Books

### CODE REVIEW

File: api-2-read-404s/400-and-404s.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# 400 Level Responses
```
Comment (CheezItMan) [inline api-2-read-404s/400-and-404s.md:1]: Is there supposed to be more to this file?

File: api-2-read-404s/problem-set-read-404s.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Problem Set: Read One Book, 400 and 404s
```
Comment (CheezItMan) [inline api-2-read-404s/problem-set-read-404s.md:1]: Curious how much overlap this has with the README in Hello Books.

**Review by CheezItMan — APPROVED** [review]
The content look

File: api-2-read-404s/400-and-404s.md (hunk 1 above)
Comment (beccaelenzil) [reply api-2-read-404s/400-and-404s.md:1]: working on this!

File: api-2-read-404s/problem-set-read-404s.md (hunk 2 above)
Comment (beccaelenzil) [reply api-2-read-404s/problem-set-read-404s.md:1]: empty readme

**Pull request merged**